content/zh/docs/tasks/administer-cluster/safely-drain-node.md

```diff
@@ -231,19 +232,27 @@ For a given eviction request, there are two cases:
 - 至少匹配一个预算。在这种情况下，上述三种回答中的任何一种都可能适用。
 
 <!-- 
-In some cases, an application may reach a broken state where it will never return anything
-other than 429 or 500. This can happen, for example, if the replacement pod created by the
-application's controller does not become ready, or if the last pod evicted has a very long
-termination grace period.
+## Stuck evictions
+
+In some cases, an application may reach a broken state, one where unless you intervene the
+eviction API will never return anything other than 429 or 500.
+
+For example: this can happen if ReplicaSet is creating Pods for your application but
+the replacement Pods do not become `Ready`. You can also see similar symptoms if the
+last Pod evicted has a very long termination grace period.
 
 In this case, there are two potential solutions:
 
-- Abort or pause the automated operation. Investigate the reason for the stuck application, and restart the automation.
-- After a suitably long wait, `DELETE` the pod instead of using the eviction API.
+- Abort or pause the automated operation. Investigate the reason for the stuck application,
```

Comment:
could you sync there changes ?

Reply:
ok, I will sync those changes.

Reply:
I've synced the changes. @tanjunchen  Thanks for the review.